src/assert_obj_safe.rs

```diff
@@ -71,6 +63,6 @@
 #[macro_export]
 macro_rules! assert_obj_safe {
     ($($xs:path),+ $(,)?) => {
-        $(const _: Option<&$xs> = None;)+
+        $(const _: Option<&dyn $xs> = None;)+
```

Comment:
This is only a cosmetic change. The `dyn` syntax for trait objects exists since Rust 1.27 (the MSRV of this crate is 1.37) and the old syntax of just using the trait name is now deprecated.